### DISCUSSION TITLE
Issues with Figures and Divs for PDF rendering

### DISCUSSION THREAD

**Discussion by friosavila** [quarto-dev/quarto-cli#11632] [Q&A]
### Description

Im currently using Quarto 1.6.39

For a while, I have been using the following code to wrap tables and figures, so I could create a consistent setup to post tables and figures on PDF and HTML

```
:::{#tbl-f2}
![caption](tidyplot_test.png)
:::
```

However, now that im doing that it gives me an error associated with \centering.
Seems that when creating the latex, its creating the following:

```
\begin{figure}[H]

\centering{

\begin{figure}
\centering
\pandocbounded{\includegraphics[keepaspectratio]{Number of Children.png}}
\caption{Distribution of the number of children between 0 and 12}
\end{figure}

}

\caption{\label{fig-myfig1}}

\end{figure}%
```

In other words, trying to create a figure within a figure. Thus breaking.

Is there a work around the problem?

The simplified version of the code is

```
---
title: "Untitled"
format: pdf
---


:::{#tbl-f2}
![caption](tidyplot_test.png)
:::
```

**Comment by mcanouil** [comment]
The correct syntax is the following as documented: https://quarto.org/docs/authoring/cross-references-divs.html

I don't believe a mix of the inline and cross-reference div syntax is documented anywhere, but maybe I missed it. If so, please point to where it is as it is a mistake.

```markdown

---
title: "Untitled"
format: pdf
---


:::{#tbl-f2}
![](tidyplot_test.png)

caption
::: 
```


In which version did the mix syntax work precisely?
